server/src/utils/cosmosClient.ts

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { CosmosClient, CosmosClientOptions, Database, Container, ContainerRequest } from '@azure/cosmos';
+import { DefaultAzureCredential } from '@azure/identity';
+
+const defaultAzureCredential = new DefaultAzureCredential();
+
+const cosmosClientOptions: CosmosClientOptions = {
+  endpoint: process.env.VV_COSMOS_DB_ENDPOINT!,
+  aadCredentials: defaultAzureCredential
+};
+
+const initializeCosmosClient = () => {
+  if (process.env.VV_COSMOS_DB_CONNECTION_STRING) {
+    return new CosmosClient(process.env.VV_COSMOS_DB_CONNECTION_STRING);
```

Comment:
This instance should be encapsulated inside this class. It should not be exposed outside

Reply:
Not sure if this is exposed but I can change CosmosClient to be class.